### PR TITLE
Finishing up cardinality traits

### PR DESCRIPTION
**Pull request by david20000813**
This is a somewhat frivolous PR, but I went through the traits for various cardinalities and found the spaces that should have cardinality traits assigned but not. This is the result of that, together with two extra things I noticed and added for those spaces: S174 is submetrizable and S1103 does not have P191.

### CODE REVIEW

File: spaces/S000174/properties/P000112.md
Comment (StevenClontz) [inline spaces/S000174/properties/P000112.md:2]: To add proofs for this space, we should add the details of its construction to the space's description, at least enough details to verify anything we prove beyond "see this citation".

Comment (StevenClontz) [reply spaces/S000174/properties/P000112.md:2]: Similar for S179

File: spaces/S001103/properties/P000191.md
Comment (StevenClontz) [inline spaces/S001103/properties/P000191.md:3]: This property is hereditary: can we defer to a smaller power with a non G delta point?

**Review by StevenClontz — CHANGES_REQUESTED** [review]
Thanks! See my comments and let me know what you think.

File: spaces/S000174/properties/P000112.md
Comment (david20000813) [reply spaces/S000174/properties/P000112.md:2]: Thanks for the suggestion. Both constructions are somewhat long, so I might need to skip some details. I'll let you review once I'm done with that.

File: spaces/S001103/properties/P000191.md
Comment (david20000813) [reply spaces/S001103/properties/P000191.md:3]: Ah, in that case I can add lack of P191 to the continuum power of countable discrete space (S101), for which P191 is currently unknown, and then just reference that for S1103. (Though, somewhat off-topic, do you know why the numbering is 1103 for this space?)

Comment (StevenClontz) [reply spaces/S001103/properties/P000191.md:3]: https://github.com/pi-base/data/commit/77e6ee1fc93ab735f0d9cfc429daf09a9177d86a

It's space 103 in *Counterexamples* and long ago there was an attempt to match our IDs to that, though there was a mix up making everything usually a few off. But calling it S001103 was an attempt to keep the connection to 103.

Comment (StevenClontz) [reply spaces/S001103/properties/P000191.md:3]: Yeah - I think it *was* S103, but somewhere our numbering got changed, removing the connections of pi-Base space IDs to IDs from *Counterexamples*. 😢

**Comment by david20000813** [comment]
@StevenClontz Hi Steven, all your suggestions have been implemented. Please take a look when you have time.

File: spaces/S000174/README.md
Comment (StevenClontz) [inline spaces/S000174/README.md:9]: For simplicity, could this starting space be a discrete space of cardinality $\aleph_1$? (Such a space isn't listed in pi-Base unless you assume the continuum hypothesis, I'm realizing.)

Generally we reserve $X$ for the space being defined; rather than re-topologizing $X$, perhaps we can call the starting space $Y$ (or $\omega_1$?) then define a topology for $X$ without "overwriting" the previous $X$?

Comment (StevenClontz) [reply spaces/S000174/README.md:9]: I think I'm missing something here; if $X$ is discrete $\omega_1$, then $X_\xi$ could be $\xi\subseteq\omega_1$, but then I think we recover the original discrete topology through this process.

Comment (david20000813) [reply spaces/S000174/README.md:9]: @StevenClontz I'll change $X$ to $Y$, but yeah, that is weird. I thought I was copying the paper's construction nearly verbatim, but perhaps I missed something. Let me check again.

Comment (david20000813) [reply spaces/S000174/README.md:9]: Oh, I realized what went wrong. The general construction is described as I stated. And the resulting $X$ will be $T_6$, locally metrizable, and collectionwise normal, regardless of what $X$ we started with is. But the final property, non-paracompact, that's asserted in the paper, only applies to $X$ with some specific property. The paper didn't mention those additional requirements until the proposition asserting non-paracompactness, so I just missed that. I'll add that in.

Comment (david20000813) [reply spaces/S000174/README.md:9]: Basically, one has to ensure that each $X_\xi$ actually adds something, i.e., $X_\xi \setminus \left(\bigcup_{\alpha<\xi} X_\alpha\right) \neq \varnothing$, even for limit ordinals $\xi$. And this is not true for discrete space $\omega_1$. The example given in the paper is instead $\omega_1^\omega$, where $\omega_1$ is equipped with the discrete topology. I'll just add in this specific example for concreteness.

**Comment by david20000813** [comment]
@StevenClontz For concreteness, I replaced the general construction of S174 outlined in the paper for the specific example the paper used.

File: spaces/S000174/properties/P000163.md
Comment (StevenClontz) [inline spaces/S000174/properties/P000163.md:10]: It'd be nice to have a reference for $|X|\leq|w(X)|^\omega$ for metrizable spaces; it sounds right but it's not something I would spout off-hand if asked.

**Review by StevenClontz — COMMENTED** [review]
Thanks again for tackling this chore. I think this is ready once we have a reference for the cardinal inequality I noted. (I spent a few minutes and will spend a few more trying to dig one up myself.)

File: spaces/S000174/properties/P000163.md
Comment (david20000813) [reply spaces/S000174/properties/P000163.md:10]: @StevenClontz I'm not sure whether the general case has been explicitly written down anywhere, but in this specific case it is effectively already mentioned in [T404](https://topology.pi-base.org/theorems/T404), where it is noted that for a sequential, US space $X$, if it has density character at most continuum, then its cardinality is at most continuum as well. The same argument cited in T404 can be generalized to show that $|X| \leq \chi(X)^\omega$ when $X$ is sequential and US, where $\chi(X)$ is the density character of $X$. And, of course, $\chi(X) \leq w(X)$.

Comment (david20000813) [reply spaces/S000174/properties/P000163.md:10]: @StevenClontz I'll just cite the MSE answer referenced in T404 for this.

Comment (StevenClontz) [reply spaces/S000174/properties/P000163.md:10]: Oh, I should have remembered T404 since I participated in those Math SE comments. 😅

If it's relevant for your update, note that $d(X)$ is more common for density, while $\chi(X)$ is used for the supremum of local weights. https://en.wikipedia.org/wiki/Cardinal_function

I think this is really telling us we need to add $d(X)\leq\mathfrak c$ as a property, and update T404. That can be done after this.

**Comment by david20000813** [comment]
@StevenClontz I've made the needed changes now.

**Comment by david20000813** [comment]
Ah, before approving and merging, I just found out there are some CRLF issues. Let me correct those first.

**Comment by david20000813** [comment]
Done.

File: spaces/S000174/properties/P000163.md
Comment (StevenClontz) [inline spaces/S000174/properties/P000163.md:7]: I'd prefer to see more here (e.g. T404's Math StackExchange reference). But I think it's best to get everything merged in; I'll propose my own PR for a tweak or two for the community to consider.

**Review by StevenClontz — APPROVED** [review]

File: spaces/S000174/properties/P000163.md
Comment (StevenClontz) [reply spaces/S000174/properties/P000163.md:7]: For posterity - this was perfectly fine, since the underlying set is now $\omega_1^\omega$.

**Pull request merged**
